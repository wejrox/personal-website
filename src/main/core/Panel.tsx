import * as React from "react";
import Row from "reactstrap/lib/Row";
import Col from "reactstrap/lib/Col";

/**
 * A panel that displays content formatted based on the device viewed on and it's proportions.
 */
class Panel extends React.Component {

    public render() {
        return (
            <Row>
                <Col>
                    Panel
                </Col>
            </Row>
        );
    }
}

export default Panel;
