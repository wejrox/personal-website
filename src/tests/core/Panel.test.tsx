import * as Enzyme from "enzyme";
import * as React from "react";
import Panel from "../../main/core/Panel";

describe("Panel Component", () => {
    const wrapper = Enzyme.shallow(<Panel/>);

    it("renders correctly", () => {
        expect(wrapper).toMatchSnapshot();
    });
});
