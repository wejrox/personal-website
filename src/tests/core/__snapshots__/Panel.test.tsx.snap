// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Panel Component renders correctly 1`] = `ShallowWrapper {}`;
